_content/articles/wat-phum-thmei-palm-leaf-library_menchaca-philip.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Cambodia’s Wat Phum Thmei Palm Leaf Library and the Resilience of Buddhist Texts"
+authors:
+  - "Philip Menchaca"
+source_url: "https://tricycle.org/article/wat-phum-thmei-texts/#:~:text=Importantly%2C%20the%20Wat%20Phum%20Thmei,complex%20cultural%20and%20spiritual%20landscape."
+drive_links:
```

Comment:
Yes, for cases like this where we might feel shy about sharing the PDF, you can rename "drive_links" to "hidden_links" 😊